client/jetpack-connect/controller.js

```diff
@@ -486,3 +487,9 @@ export function redirectToSiteLessCheckout( context, next ) {
 
 	next();
 }
+
+export function redirectToCloudPricingPage() {
+	window.location.href = JETPACK_PRICING_PAGE;
+
+	return;
```

Comment:
Not a blocker - I don't think the return is needed here?

Reply:
I'm used to explicitly add `return`s in route middlewares to prevent any code running after a redirect, but I'm indeed not calling `next` here so I'll remove it.